src/pages/patterns/drag-and-drop.mdx

```diff
@@ -0,0 +1,596 @@
+---
+title: Drag and drop
+description: >
+  Drag and drop pattern is often used to change the order of items in a list or to move and organize items between lists.
+---
+
+![List of items with one item being dragged to show parts of the item.](figma:patterns-dnd-intro.png)
+
+## Anatomy
+
+[Drag and drop](https://zendeskgarden.github.io/react-components/?path=/story/packages-drag-drop-patterns--drag-and-drop)
```

Comment:
note for @geotrev: this and other pattern page URLs will need to be updated when the component page lands.